example/dev-env-setup/roles/start-rackhd/files/rackhd.yml

```diff
@@ -0,0 +1,21 @@
+apps:
```

Comment:
You have two copies of rackhd.yml, another one is:
example/dev-env-setup/roles/pm2/files/rackhd.yml